### ISSUE TITLE
how  run multi GPU

### ISSUE THREAD

**Issue by ptklx** [qubvel/segmentation_models.pytorch#218]
i use nn.DataParallel() 
but error

**Comment by howard-mahe** [comment]
Hello,

I can't provide a short answer because I haven't try it yet. Also, I can't help on your error because you didn't provide any details.

However, you could find inspiration in [HRNet-Semantic-Segmentation (branch pytorch-v1.1)](https://github.com/HRNet/HRNet-Semantic-Segmentation/tree/pytorch-v1.1) project for multi-gpu implementation. If you want to keep fine-tuning your BatchNormalisation layers, you will likely need to use the official [nn.SyncBatchNorm](https://pytorch.org/docs/master/generated/torch.nn.SyncBatchNorm.html#torch.nn.SyncBatchNorm) or the memory-efficient [InplaceABN](https://github.com/mapillary/inplace_abn).
